builder/plugins/levels-parse.js

```diff
@@ -0,0 +1,15 @@
+const tags = require('metalsmith-tags');
+
+module.exports = (path) => (files, metalsmith, done) => tags({
```

Comment:
Add a comment explaining why you need this file

Reply:
done